src/styles/blocks/mixins.scss

```diff
@@ -0,0 +1,7 @@
+@mixin transition-hover($property, $value) {
+  transition: #{$property} 0.3s;
```

Comment:
also you use 0.3s for transition in different places, I think it would be a good idea to keep this value in variable

don't forget to update this value-variable in your scss files